app/src/store/wallet.ts

```diff
@@ -41,6 +41,7 @@ const defaultProvider = new JsonRpcProvider(RPC_URL);
 let onboard: OnboardAPI; // instance of Blocknative's onboard.js library
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const rawProvider = ref<any>(); // raw provider from the user's wallet, e.g. EIP-1193 provider
+const defProvider = ref<Web3Provider | JsonRpcProvider>(markRaw(defaultProvider)); // default provider
```

Comment:
Let's export `DEFAULT_PROVIDER` from `chains.ts` instead of defining it in `wallet.ts` and exporting it as a ref, since it will be a constant